.github/labeler.yml

```diff
@@ -4,7 +4,7 @@ kind/docs:
 
 kind/ui:
   - "web_src/**/*"
-  - all: ["templates/**", "!templates/swagger/v1_json.tmpl"]
+  - any: ["templates/**", "!templates/swagger/v1_json.tmpl"]
```

Comment:
No, `any` is certainly not correct. It would also fire on `docs/**`.

Reply:
Why should it fire on the docs, which are on another path?

Reply:
Because you are explicitly instructing it to accept `any modified file if it is inside 'templates' or not the swagger file`.
Every file except the swagger file is not a swagger file.
Furthermore, as it is `any`, even the swagger file will be marked as `ui` as it is inside the templates.

At least that's how I as a noob regarding this particular action interpret this statement.
It is of course possible that there is a directory restriction, but if that's the case then that is rather non-obvious.